src/Paket.Core/Installation/InstallProcess.fs

```diff
@@ -293,20 +296,31 @@ let private applyBindingRedirects isFirstGroup createNewBindingFiles cleanBindin
                 librariesForPackage
                 |> Seq.choose(fun (library,redirects,profile) ->
                     try
-                        let assembly = Mono.Cecil.AssemblyDefinition.ReadAssembly(library.Path)
-                        Some (assembly, BindingRedirects.getPublicKeyToken assembly, assembly.MainModule.AssemblyReferences, redirects, profile)
+                        let assemblyName = AssemblyName.GetAssemblyName(library.Path)
+                        let publicKeyToken = AssemblyMetadata.getPublicKeyToken assemblyName
+                        let metadataName = AssemblyMetadata.AssemblyName assemblyName
+                        let references = 
+                            match AssemblyMetadata.getAssemblyReferences metadataName with
+                            | Trial.Pass list -> list
+                            | Trial.Warn(list, warn) -> 
+                                warn |> List.iter (fun ex -> ex |> string |> traceWarn)
+                                list
+                            | Trial.Fail exns ->
+                                exns |> List.iter (fun ex -> ex |> string |> traceError)
```

Comment:
Depending on the scenarios where this happens (traceError and traceWarn) we might need to restrict this to verbose mode later.
Any idea when this happens?